Musician Module/MusicianModule.cs

```diff
@@ -0,0 +1,512 @@
+        private const string DD_TITLE = "Title";
+        private const string DD_ARTIST = "Artist";
+        private const string DD_USER = "User";
+        private const string DD_HARP = "Harp";
+        private const string DD_FLUTE = "Flute";
+        private const string DD_LUTE = "Lute";
+        private const string DD_HORN = "Horn";
+        private const string DD_BASS = "Bass";
+        private const string DD_BELL = "Bell";
+        private const string DD_BELL2 = "Bell2";
+        private List<string> Instruments = new List<string>{
+           "Harp", "Flute", "Lute", "Horn", "Bell", "Bell2", "Bass"
+        };
+        private WindowTab MusicianTab;
+        private MusicPlayer MusicPlayer;
+        private HealthPoolButton StopButton;
+        private XmlMusicSheetReader xmlParser;
+        private List<SheetButton> displayedSheets;
+        private List<RawMusicSheet> Sheets;
+
+        /// <summary>
+        /// Ideally you should keep the constructor as is.
+        /// Use <see cref="Initialize"/> to handle initializing the module.
+        /// </summary>
+        [ImportingConstructor]
+        public Musician_Module([Import("ModuleParameters")] ModuleParameters moduleParameters) : base(moduleParameters) { /* NOOP */ }
+
+        /// <summary>
+        /// Define the settings you would like to use in your module.  Settings are persistent
+        /// between updates to both Blish HUD and your module.
+        /// </summary>
+        protected override void DefineSettings(SettingsManager settingsManager)
+        {
+        }
+
+        /// <summary>
+        /// Allows your module to perform any initialization it needs before starting to run.
+        /// Please note that Initialize is NOT asynchronous and will block Blish HUD's update
+        /// and render loop, so be sure to not do anything here that takes too long.
+        /// </summary>
+        protected override void Initialize()
+        {
+            Directory.CreateDirectory(Path.Combine(GameService.FileSrv.BasePath, "musician"));
+            xmlParser = new XmlMusicSheetReader();
+            displayedSheets = new List<SheetButton>();
+        }
+
+        /// <summary>
+        /// Load content and more here. This call is asynchronous, so it is a good time to
+        /// run any long running steps for your module. Be careful when instancing
+        /// <see cref="Blish_HUD.Entities.Entity"/> and <see cref="Blish_HUD.Controls.Control"/>.
+        /// Setting their parent is not thread-safe and can cause the application to crash.
+        /// You will want to queue them to add later while on the main thread or in a delegate queued
+        /// with <see cref="Blish_HUD.DirectorService.QueueMainThreadUpdate(Action{GameTime})"/>.
+        /// </summary>
+        protected override async Task LoadAsync()
+        {
+            MusicianTab = GameService.Director.BlishHudWindow.AddTab("Musician", ICON, BuildHomePanel(GameService.Director.BlishHudWindow), 0);
```

Comment:
LoadAsync runs async, so the call to add the new tab in the Blish HUD window has the potential to throw an exception.  I think it's unlikely, but it's very possible.  Might be best to build the panels for your tab here and then in `OnModuleLoaded` actually add the tab to the Blish HUD window.